src/Game.ts

```diff
@@ -18,6 +20,9 @@ class Game {
         this.blackClock.render();
         this.whiteClock = new Clock('white', 15, 0, 'whiteClock');
         this.whiteClock.render();
+        this.gameOverScreen = new GameOver();
+        this.gameOverScreen.gameOverScreenTrigger();
+        this.gameOverScreen.render();
```

Comment:
Potrzebujemy funkcji która będzie sprawdzała czy jest zakończenie gry i jeżeli true to wtedy to się powinno renderować